viseron/__init__.py

```diff
@@ -200,6 +201,7 @@ class Viseron:
 
     def __init__(self) -> None:
         self.states = States(self)
+        self.compreface_recognition: RecognitionService = None
```

Comment:
The RecogntionService instance is better stored in vis.data to keep the Viseron object clean of any component specific declaractions.
As an example you can have a look at the `edgetpu` component:
https://github.com/roflcoopter/viseron/blob/379b7a3ab8645a0448cd7ca89c3ff79da5b6500a/viseron/components/edgetpu/object_detector.py#L26-L30

Reply:
This has been addressed in my second commit